### ISSUE TITLE
Duration.js should handle Infinity

### ISSUE THREAD

**Issue by derhuerst** [icholy/Duration.js#11]
```
d = new Duration(Infinity)
d.toString()  // InfinitydNaNhNaNmNaNsNaNms
```

**Comment by icholy** [comment]
What would the correct behaviour be?

**Comment by derhuerst** [comment]
Well it should serialize correctly to something like `'∞ms'`. At least, there shall not occur any `Nan` value.

**Comment by icholy** [comment]
I thought about this for a bit, and I don't think an infinite duration makes sense. I'm classifying this under user error. I'll add a check so that it refuses to accept such values.

**Comment by icholy** [comment]
Just published version 3.5.0
